plugins/irma-web/dom-manipulations.js

```diff
@@ -109,7 +109,9 @@ module.exports = class DOMManipulations {
   _stateShowingIrmaButton() {
     return `
       <!-- State: ShowingButton -->
-      <button class="irma-web-button" data-irma-glue-transition="openIrmaApp">${this._translations.button}</button>
+      <a id="irma-web-button-link" data-irma-glue-transition="openIrmaApp">
+        <button class="irma-web-button">${this._translations.button}</button>
+      </a>
```

Comment:
I'm guessing that you're still using the button because my styles don't account for links with the `irma-web-button` class..? We can fix that in the css.

Reply:
Yes exactly, I tried fixing it in the CSS, but I'm not really a hero in CSS and I wasn't able to make it look nicely. If you're good in CSS you may of course try.

Reply:
Will take a look